src/common/helpers/imageHelpers.ts

```diff
@@ -96,9 +97,22 @@ export class CloudinaryHelper {
         }
         cldImage.format("auto").quality("auto");
         let imageUrl = cldImage.toURL();
+
         // for cases that the image is fetched from a remote url keep serving it as fetch instead of upload
         if (this.cloudinaryFetchUrlRegex.test(image.url)) {
             imageUrl = imageUrl.replace("/upload/", "/fetch/");
+            let queryString = "";
+            try {
+                // fetch images might have a query string in the external url which is removed by cloudinary
+                // needs to be added after encoding to avoid errors
+                const urlParts = image.url.split(this.cloudinaryFetchUrlRegex);
+                const originalUrl = urlParts[urlParts.length - 1];
```

Comment:
`const originalUrl = image.url.split(this.cloudinaryFetchUrlRegex).pop();` ?
or if you want to keep it separated just
`const originalUrl = urlParts.pop();`